nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/main/java/org/apache/nifi/atlas/hook/NotificationSender.java

```diff
@@ -350,10 +351,10 @@ void send(final List<HookNotification.HookNotificationMessage> messages, final C
                 }
             });
 
-            if (refQualifiedName == null) {
+            if (typedQualifiedName == null) {
                 return null;
             }
-            return new Tuple<>(refQualifiedName, typedQualifiedNameToRef.get(toTypedQualifiedName(typeName, refQualifiedName)));
+            return new Tuple<>(typedQualifiedName, typedQualifiedNameToRef.get(toTypedQualifiedName(typeName, typedQualifiedName)));
```

Comment:
Calling `toTypeQualifiedName()` not needed here, `typeQualifiedName` variable already contains the type + qualified name.